hub-search-proxy/handler.js

```diff
@@ -18,6 +18,8 @@ const deliveryApiClient = contentful.createClient({
   accessToken: token
 })
 
+const VALID_CONTENT_TYPES = ['article','caseStudy','equipment','event', 'funding', 'service','software','subHub'];
```

Comment:
@rosemcc in the previous version the content types were called "casestudy" and "subhubs". Just checking if this change is intended?

Reply:
ooh, good catch - I actually need to handle lower casing during the bulk upload. Thank you!